.bazelrc

```diff
@@ -1,5 +1,3 @@
-# For compatibility with Bazel 0.27 through 0.29. See:
```

Comment:
A bit confusing to remove these options as part of this PR - I've factored them out into https://github.com/tensorflow/tensorboard/pull/3535

Reply:
Thanks for doing that!